src/stories/gutters.css

```diff
@@ -0,0 +1,7 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+$gutterWidth__xxLarge: 2.4rem;
+$gutterWidth__xLarge: 2rem;
+$gutterWidth__large: 1.6rem;
+$gutterWidth__medium: 1.2rem;
+$gutterWidth__small: 0.8rem;
+$gutterWidth__xSmall: 0.4rem;
```

Comment:
Why is this file here?

Reply:
no idea

Reply:
Maybe it was an accidental drag-drop on your editor? Feel free to kill it.